security.rst

```diff
@@ -1838,30 +1837,28 @@ To enable logging out, activate the  ``logout`` config parameter under your fire
             $mainFirewall = $security->firewall('main');
             // ...
             $mainFirewall->logout()
-                // the argument can be either a route name or a path
-                ->path('app_logout')
+                ->path('/logout')
 
                 // where to redirect after logout
                 // ->target('app_any_route')
             ;
         };
 
-``path`` can be either a route name, or a URI.
+Symfony will then un-authenticate users navigating to the configured ``path``,
+and redirect them to the configured ``target``.
```

Comment:
```suggestion
Symfony will then un-authenticate users navigating to the configured ``path``,
and redirect them to the configured ``target``. You can generate URLs to this
path using the ``_security_<firewallname>`` route name (e.g. ``_security_main``).
```

Reply:
Shouldn’t the `LogoutUrlGenerator` be mentioned instead?